tests/tests/son_operations_tests.cpp

```diff
@@ -8,134 +8,134 @@
 using namespace graphene::chain;
 using namespace graphene::chain::test;
 
-class test_create_son_member_evaluator: public create_son_evaluator {
-public:
-   test_create_son_member_evaluator( database& link_db ):
-                                     ptr_trx_state( new transaction_evaluation_state( &link_db ) )
-   {
-       trx_state = ptr_trx_state.get();
-   }
-   std::unique_ptr<transaction_evaluation_state> ptr_trx_state;
-};
+BOOST_FIXTURE_TEST_SUITE( son_operation_tests, database_fixture )
 
-class test_update_son_member_evaluator: public update_son_evaluator {
-public:
-   test_update_son_member_evaluator( database& link_db ):
-                                     ptr_trx_state( new transaction_evaluation_state( &link_db ) )
-   {
-      trx_state = ptr_trx_state.get();
+BOOST_AUTO_TEST_CASE( create_son_test ) {
+   generate_blocks(HARDFORK_SON_TIME);
+   while (db.head_block_time() <= HARDFORK_SON_TIME) {
+      generate_block();
    }
-   std::unique_ptr<transaction_evaluation_state> ptr_trx_state;
-};
+   generate_block();
+   set_expiration(db, trx);
+
+   ACTORS((alice)(bob));
+
+   upgrade_to_lifetime_member(alice);
+   upgrade_to_lifetime_member(bob);
 
-class test_delete_son_member_evaluator: public delete_son_evaluator {
-public:
-   test_delete_son_member_evaluator( database& link_db ):
-                                     ptr_trx_state( new transaction_evaluation_state( &link_db ) )
+   transfer( committee_account, alice_id, asset( 100000 ) );
+   transfer( committee_account, bob_id, asset( 100000 ) );
+
+   set_expiration(db, trx);
+   std::string test_url = "https://create_son_test";
+
+   // create deposit vesting
+   vesting_balance_id_type deposit;
    {
-      trx_state = ptr_trx_state.get();
+      vesting_balance_create_operation op;
+      op.creator = alice_id;
+      op.owner = alice_id;
+      op.amount = asset(10);
+      op.balance_type = vesting_balance_type::unspecified;
```

Comment:
@oxarbitrage  : we have `normal` and `gpos` now

Reply:
not in this branch: https://github.com/peerplays-network/peerplays/blob/feature/SONs-base/libraries/chain/include/graphene/chain/vesting_balance_object.hpp#L149